src/content/subgames.cpp

```diff
@@ -140,8 +162,14 @@ SubgameSpec findSubgame(const std::string &id)
 	menuicon_path = getImagePath(
 			game_path + DIR_DELIM + "menu" + DIR_DELIM + "icon.png");
 #endif
-	return SubgameSpec(id, game_path, gamemod_path, mods_paths, game_name,
+
+	SubgameSpec spec(id, game_path, gamemod_path, mods_paths, game_title,
 			menuicon_path, game_author, game_release);
+
+	if (conf.exists("name") && !conf.exists("title"))
+		spec.deprecation_msgs.push_back("name in game.conf is deprecated, please use title instead");
```

Comment:
```suggestion
		spec.deprecation_msgs.push_back("\"name\" setting in game.conf is deprecated, please use \"title\" instead");
```

Reply:
Is it accurate to call it a setting? In docs, that usually refers to minetest.conf

Reply:
Technically speaking they're all files that are processed by Settings, hence the name. The file name is mentioned too, hence I don't see a conflict in terminology. Up to you.